presets/workspace/models/supported_models.yaml

```diff
@@ -35,13 +35,14 @@ models:
     type: text-generation
```

Comment:
Always separate changes to `supported_models` into their own release PR. This ensures that other changes can be merged without delays caused by the time-consuming process of building images, which can require retries (e.g. docker push fails, test fails, etc.). Additionally, isolating these changes makes it easier to track and manage preset releases.